Utilities/bootstrap

```diff
@@ -108,7 +108,7 @@ g_default_sysroot = None
 if platform.system() == 'Darwin':
     g_platform_path = subprocess.check_output(
```

Comment:
Yeah I was going to create a new variable but saw that it was unused so just appended the framework path to same variable (and forgot to rename)... will fix!